python/ola/__init__.py

```diff
@@ -0,0 +1,29 @@
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# __init__.py
+# Copyright (C) 2018 Shenghao Yang
+
+import logging
+
+"""Initialization code for the ola python library"""
```

Comment:
Codacy doesn't like this:
"String statement has no effect"
https://app.codacy.com/app/peternewman/ola/pullRequest?prid=1616358

I think it may need to be above the import, looking at some other real world examples.